sw/device/tests/chip_power_sleep_load.c

```diff
@@ -0,0 +1,377 @@
+// Copyright lowRISC contributors.
```

Comment:
since this test has a host side SV component, it can only run in the DV environment. It should be moved to `sw/device/tests/sim_dv`

Reply:
This test is self contained.
The SV component was added as an additional checker due to requests of reviewers of PR #15923
and as a means for providing non-default values to some S/W constants (using sw_symbol_backdoor_overwrite).
The SV does not provide any type of stimuli and it is not needed in order for the test to finish.
In the OT & Nuvoton meeting, we discussed this issue for PR #15923 with @msfschaffner and @weicaiyang and they said the test should remain in the general folder and not in sim_dv.